include/realtime_tools/realtime_server_goal_handle.h

```diff
@@ -43,106 +44,124 @@ template <class Action>
 class RealtimeServerGoalHandle
 {
 private:
-  ACTION_DEFINITION(Action);
 
-  typedef actionlib::ServerGoalHandle<Action> GoalHandle;
+  typedef rclcpp_action::ServerGoalHandle<Action> GoalHandle;
```

Comment:
I'd personally prefer the one below due to style but up to you
```suggestion
  using GoalHandle = rclcpp_action::ServerGoalHandle<Action>;
```